public/docs/_examples/animations/ts/app/hero-list-multistep.component.ts

```diff
@@ -0,0 +1,50 @@
+import {
```

Comment:
Do we have a Table of Contents for these examples with titles, purpose and differences columns ?